client/app/components/Technologies/index.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import React from 'react';
+import style from './style.module.scss';
+
+const technologies = [
+    'android',
+    'angular',
+    'aws',
+    'azure',
+    'c-sharp',
+    'css',
+    'google-cloud',
+    'html',
+    'ios',
+    'java',
+    'javascript',
+    'kotlin',
+    'mongoDB',
+    'mySQL',
+    'node-js',
+    'oracle',
+    'postgreSQL',
+    'python',
+    'react-native',
+    'react',
+    'typescript',
+];
+
+const Technologies = () => {
+    return (
+        <div className={style.technologiesBar}>
+            <h2 className={style.technologiesTitle}>Technologies</h2>
+            <div className={style.logos}>
+                {technologies.map((tech, index) => (
+                    <img
+                        key={index}
+                        src={`/assets/tech-logos/${tech}.svg`}
```

Comment:
si path-ul asta poate fi scos intr-o constanta la fel ca file extension. vrem sa fie cat mai modulara aplicatia.

poti sa faci in fisierul de contants un fisier .ts nou numit technologies cosntants ca sa le gasim mai usor si sa pui aici si TECHNOLOGIES si TECHLOGIES_PATH si TECHLOGIES_FILE_EXTENSION
si ai avea la sursa src={`${TECHLOGIES_PATH}${tech}${TECHLOGIES_FILE_EXTENSION}`}